stacks/linkerd2/upgrade.sh

```diff
@@ -33,7 +33,10 @@ $BINARY upgrade | kubectl apply --prune -l linkerd.io/control-plane-ns=linkerd \
 # ensure services are running
 kubectl get deployments -o custom-columns=NAME:.metadata.name | tail -n +2 | while read -r line
 do
-  kubectl rollout status -w deployment/"$line"
+  if [ ! -z "$line" ]
+  then 
+    kubectl rollout status -w deployment/"$line"
+  fi
```

Comment:
Maybe we need an else statement with an error drop?

Reply:
hmmmm

Reply:
thank you, added, but I couldn't test it. 🤞🏿